### PR TITLE
Copy urls file to repos location

### PR DESCRIPTION
**Pull request by AnonymousWalker**
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/82)

### CODE REVIEW

File: fetcher-pipeline/Dockerfile [hunk 1]
```diff
@@ -17,4 +17,6 @@ RUN curl -O https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x
     tar zxvf openjdk-11.0.2_linux-x64_bin.tar.gz && mv jdk-11* /usr/local/
 
 COPY entrypoint.sh /entrypoint.sh
+COPY gl_repo_urls.txt $ORATURE_REPO_DIR
```
Comment (danparisd) [inline fetcher-pipeline/Dockerfile:20]: I think $ORATURE_REPO_DIR should be replaced with an absolute path in the container that your script references by absolute path inside the container.

**Review by danparisd — CHANGES_REQUESTED** [review]

File: fetcher-pipeline/Dockerfile (hunk 1 above)
Comment (AnonymousWalker) [reply fetcher-pipeline/Dockerfile:20]: the python script also uses the env var to work with repo directory

`        self.repo_dir = os.getenv("ORATURE_REPO_DIR")`

**Review by mXaln — COMMENTED** [review]
> Reviewed 1 of 3 files at r3, 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/82)* status: all files reviewed, 1 unresolved discussion (waiting on @danparisd)

___
*[fetcher-pipeline/Dockerfile, line 20 at r3](https://reviewable.io/reviews/bible-translation-tools/fetcher/82#-MMXDizAeOKxL4WOAXGp-r3-20:-MMXImf8EnX-JX6MFjmj:boa565m) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/4c75c9ef9f083258f1efddfb5e4653d827c7034c/fetcher-pipeline/Dockerfile#L20)):*
<details><summary><i>Previously, AnonymousWalker (Tony) wrote…</i></summary><blockquote>

the python script also uses the env var to work with repo directory, same as fetcher does.

`        self.repo_dir = os.getenv("ORATURE_REPO_DIR")`

</blockquote></details>

I'm not sure if you can use env var in dockerfile this way. You would need to use ARG directive. And then add it to docker-compose.yaml file in `args` section. 

https://vsupalov.com/docker-env-vars/ 

@danparisd might know better that me

File: fetcher-pipeline/Dockerfile (hunk 1 above)
Comment (danparisd) [reply fetcher-pipeline/Dockerfile:20]: Dockerfile is different. We're not passing env vars in at build time. env vars are for runtime. This is a statically loaded file at build time so an absolute path is advised. I'd recommend treating the repo list and the temp dir as separate.

**Review by danparisd — CHANGES_REQUESTED** [review]

File: fetcher-pipeline/Dockerfile (hunk 1 above)
Comment (AnonymousWalker) [reply fetcher-pipeline/Dockerfile:20]: okay sure. I've made the change.

**Review by mXaln — APPROVED** [review]
> Reviewed 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/82)* status: all files reviewed, 1 unresolved discussion (waiting on @danparisd)

File: fetcher-pipeline/Dockerfile (hunk 1 above)
Comment (danparisd) [reply fetcher-pipeline/Dockerfile]: looks good

**Review by danparisd — APPROVED** [review]
lgtm

**Pull request merged**